ecosystem/typescript/sdk/package.json

```diff
@@ -54,5 +54,5 @@
     "tweetnacl": "^1.0.3",
     "yarn": "^1.22.18"
   },
-  "version": "0.0.20"
+  "version": "1.0.0"
```

Comment:
wow, not even 0.1.0? 1.0 seems to be better reserved for our mainnet?

Reply:
In semver it seems that only the top level number is for breaking changes